esp-wifi/MIGRATING-0.10.md

```diff
@@ -42,3 +42,42 @@ The cost of this is that we need to rename the various `async` methods on `WifiC
 - controller.start().await.unwrap();
 + controller.start_async().await.unwrap();
 ```
+
+## The blocking networking stack was removed
+
+The blocking networking stack is not included anymore. You can use e.g. `smoltcp-nal` or even use `smoltcp` directly.
+
+For an easy migration path there is https://github.com/bjoernQ/blocking-network-stack.git which is basically the previously included networking stack as it's
+own crate.
+
+The `create_network_interface` function doesn't take `&mut SocketSet[..]` anymore.
+
+```diff
++use blocking_network_stack::Stack;
+use esp_wifi::{
+-    wifi_interface::WifiStack,
+};
+
+-    let init = init(
+-        timg0.timer0,
+-        Rng::new(peripherals.RNG),
+-        peripherals.RADIO_CLK,
+-    )
+-    .unwrap();
++    let mut rng = Rng::new(peripherals.RNG);
++    let init = init(timg0.timer0, rng.clone(), peripherals.RADIO_CLK).unwrap();
+ 
+     let mut wifi = peripherals.WIFI;
++    let (iface, device, mut controller) =
++        create_network_interface(&init, &mut wifi, WifiStaDevice).unwrap();
++
+-    let (iface, device, mut controller, sockets) =
+-        create_network_interface(&init, &mut wifi, WifiStaDevice, &mut socket_set_entries).unwrap();
++    let mut socket_set_entries: [SocketStorage; 3] = Default::default();
```

Comment:
the line is present before *and* after right? In either version this array was still required. Which means you don't need the `+` here, and it should be moved above `create_network_interface`